dipy/workflows/tests/test_tracking.py

```diff
@@ -143,6 +143,16 @@ def test_local_fiber_tracking_workflow():
             lf_track_pam.last_generated_outputs['out_tractogram']
         assert_false(is_tractogram_empty(tractogram_path))
 
+        # Test tracking with binary tissue classifier
+        lf_track_pam = LocalFiberTrackingPAMFlow()
+        lf_track_pam._force_overwrite = True
+        lf_track_pam.run(pam_path, gfa_path, seeds_path,
```

Comment:
Although this is fine for testing, using a binary tissue classifier on the GFA will result in using a mask with 1s everywhere as the GFA > 0 in all voxels. I suggest we use the threholded gfa mask defined for the seeding instead.

Here replacing `gfa_path`, with `mask_path` and setting at line 131:
```mask_path = mask_flow.last_generated_outputs['out_mask']```

Reply:
Thanks for the review @gabknight. Done in b3a4b27.